package Message;


message PaymentVerifyRequest {
	required string receiptData = 1; // 凭证
	required int64 playerid = 2;
}

message PaymentVerifyResponse {
	required int32 result = 1;
	optional int64 player_id = 2;
	optional string product_id = 3;
	optional string transaction_id = 4;
}

message PaymentSaveTransactionRequest {
	required string transaction_id = 1; 
	required string product_id = 2; 
	required int64 player_id	= 3;
}

message PaymentSaveTransactionResponse {
	required int32 result		= 1;
	optional int64 player_id	= 2;
	optional string product_id	= 3;
}

